addons/im_livechat/controllers/chatbot_script.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import http
+from odoo.http import request
+from odoo.tools import is_html_empty, plaintext2html
+
+
+class LivechatChatbotScriptController(http.Controller):
+    @http.route('/im_livechat/chatbot_restart', type="json", auth="public", cors="*")
+    def im_livechat_chatbot_restart(self, channel_uuid, chatbot_script_id):
+
+        mail_channel = request.env['mail.channel'].sudo().search([('uuid', '=', channel_uuid)], limit=1)
+        chatbot = request.env['chatbot.script'].browse(chatbot_script_id)
+        if not mail_channel or not chatbot.exists():
+            return False
```

Comment:
Don't return `False` from JSON endpoint, json supports `None` (`null`) (even more so as it returns `None` at the end if there's no posted message anyway).